packages/cli/test/integration.test.ts

```diff
@@ -2770,20 +2648,20 @@ test('should show prompts to set up project during first deploy', async () => {
   const output = await now;
 
   // Ensure the exit code is right
-  expect(output.exitCode).toBe(0);
+  expect(output.exitCode, formatOutput(output)).toBe(0);
 
   // Ensure .gitignore is created
   const gitignore = await readFile(path.join(dir, '.gitignore'), 'utf8');
   expect(gitignore).toBe('.vercel\n');
 
   // Ensure .vercel/project.json and .vercel/README.txt are created
-  expect(await exists(path.join(dir, '.vercel', 'project.json'))).toBe(true);
-  expect(await exists(path.join(dir, '.vercel', 'README.txt'))).toBe(true);
+  expect(path.join(dir, '.vercel', 'project.json'));
+  expect(path.join(dir, '.vercel', 'README.txt'));
```

Comment:
```suggestion
  expect(await exists(path.join(dir, '.vercel', 'project.json'))).toBe(true);
  expect(await exists(path.join(dir, '.vercel', 'README.txt'))).toBe(true);
```

This seems to be a typo